src/Hive.IO/Hive.IO/Building/compile_sia2024_json.py

```diff
@@ -4,14 +4,14 @@
 import csv
 import json
 
-from yaml.loader import FullLoader
-
 # map the csv header names to Sia2024Record names...
 # (try to keep them as similar as possible)
 header_map = {    
```

Comment:
More related to your previous PR on the installer... Could you add `compile_sia2024_json.py` and `generate_sia2024_schedules_json.py`to the `build.cmd`?

Reply:
i will add this to the other PR